lib/cell.rb

```diff
@@ -30,16 +30,14 @@ def fire_upon
   def render(player = false)
     if player == true && @ship != nil && @fired_upon == false
       "S"
-    elsif fired_upon? == true && @ship != nil && @ship.sunk? == true
+    elsif @ship != nil && @ship.health == 0 && @fired_upon == true
```

Comment:
This change was another way to get to the same point. We had a few issues so we changed values and signs, even though not really changing our end goal